docs/developer/process/agent-release.md

```diff
@@ -112,4 +114,7 @@ ddev agent changelog
 ddev agent integrations
 ```
 
+See more options for [`ddev agent changelog`](../ddev/cli.md#changelog) and [`ddev agent integrations`](../ddev/cli.md#integrations).
+
+Update the contents of the [`AGENT_CHANGELOG`][agent-changelog] and [`AGENT_INTEGRATIONS`][agent-integrations] files. 
```

Comment:
Do like:

1. run A to update X
1. run B to update Y

Reply:
updated